packages/cloud-cognitive/src/components/Datagrid/styles/useExpandedRow.scss

```diff
@@ -0,0 +1,13 @@
+/*
+* Licensed Materials - Property of IBM
+* 5724-Q36
+* (c) Copyright IBM Corp. 2020 - 2021
+* US Government Users Restricted Rights - Use, duplication or disclosure
+* restricted by GSA ADP Schedule Contract with IBM Corp.
+*/
+
+@import './variables';
+
+.#{$block-class}__expanded_row_content {
+  padding: $layout-01 $layout-01 $layout-02 $layout-05;
```

Comment:
Layout tokens are deprecated — use spacing tokens.

Reply:
Fixed. I replaced the layout tokens with spacing tokens.